<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>History</class>
 <widget class="QWidget" name="History">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Command History</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="InputHistoryLimit">
        <property name="text">
         <string>Limit command history size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="inputHistoryGroup" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="commandsGroup">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Limit to:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="InputHistoryLines">
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="InputHistory_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>commands</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="OutputLimit">
        <property name="text">
         <string>Limit number of lines in output</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="outputGroup" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Limit to:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="OutputLines">
           <property name="minimum">
            <number>100</number>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Output_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>maximum lines</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Output History</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="OutputHistoryEnabled">
        <property name="text">
         <string>Save output history</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="OutputHistoryLimit">
        <property name="text">
         <string>Limit number of lines in history</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="outputHistoryGroup" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="historyGroup">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="OutputHistory_before">
           <property name="text">
            <string>Limit to:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="OutputHistoryLines">
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="OutputHistory_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>maximum lines</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>OutputHistoryEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>OutputHistoryLimit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>InputHistoryLimit</sender>
   <signal>toggled(bool)</signal>
   <receiver>inputHistoryGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OutputLimit</sender>
   <signal>toggled(bool)</signal>
   <receiver>outputGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OutputHistoryLimit</sender>
   <signal>toggled(bool)</signal>
   <receiver>outputHistoryGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OutputHistoryEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>OutputHistoryLines</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OutputHistoryEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>OutputHistory_before</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OutputHistoryEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>OutputHistory_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
